docs/stack/client/querying.md

```diff
@@ -0,0 +1,90 @@
+## Query, Observables and Subscription
+
+The entity methods provided by `@daostack/client` for querying the subgraph, by themself does not actually send the query to the server. Instead, each methods returns an *Observable* to which we can subscribe. *Subscriptions* are used so that our handler gets called every time a new value is emitted for the observable stream.
+
+Take a look at the following methods that return observable:
+```
+dao.proposals()
+proposal.state()
+```
+
+Now, in order to query the server we must subscribe
+```
+const observable = arc.daos()
+// only in the next line will a query be sent to the server
+const subscription = arc.daos.subscribe((daos) => console.log(`we found ${daos.length} results`))
+```
+
+In this guide we will describe how to query and subscribe to the subgraph that contains the index of DAOstack data.
+
+## What does Subscription do and Why we use it?
+
+By default, subscribing to an observable will do two things:
+
+1. It will send a query to the server, fetching the data
+2. It will send a subscription query to the server, which will cause the server to send you an update each time the data changes
+
+Because subscriptions can be expensive, this behavior can be controlled in several ways.
+
+First of all, we can pass Apollo's [`fetchPolicy`](https://www.apollographql.com/docs/react/api/react-apollo/#optionsfetchpolicy) argument to control how the query (from step 1) interacts with the cache:
+```
+arc.daos({}, { fetch-policy: 'cache-first'}) // the default value
+arc.daos({where: {stage: "Boosted"}}, { fetch-policy: 'network-only'}) // bypass the cache
```

Comment:
It is actually `fetchPolicy`, and there are four values:  'cache-first' | 'network-only' | 'cache-only' | 'no-cache' | 'standby' that we should describe here